vars/commonPipelineEnvironment.groovy

```diff
@@ -25,11 +41,35 @@ class commonPipelineEnvironment implements Serializable {
             return configProperties[property]
     }
 
+    def setInfluxCustomData(Map myData = [:]) {
+        influxCustomData = myData
+    }
+
+    def getInfluxCustomData() {
+        return influxCustomData
+    }
+
+    def setInfluxCustomDataMap(Map myData = [:]) {
+        influxCustomDataMap = myData
+    }
+
+    def getInfluxCustomDataMap() {
+        return influxCustomDataMap
+    }
+
     def getMtarFilePath() {
         return mtarFilePath
     }
 
     void setMtarFilePath(mtarFilePath) {
         this.mtarFilePath = mtarFilePath
     }
+
+    def setPipelineMeasurement (measurementName, value) {
+        influxCustomDataMap.pipeline_data[measurementName] = value
+    }
+
+    def getPipelineMeasurement (measurementName) {
+        return influxCustomDataMap.pipeline_data[measurementName]
```

Comment:
Does not work after calling ```setInfluxCustomDataMap()```, since the new map does not come with an entry ```pipeline_data```.

Reply:
setter is removed